### ISSUE TITLE
Convert Result Local OBX to NTE

### ISSUE THREAD

**Issue by JohnNKing** [CDCgov/trusted-intermediary#1195]
# Backlog Task

So that UCSD is able to file result ORUs that contain local codes from CDPH, OBXs need to be converted into NTE segments.

## Completion Criteria

** More refinement of this completion criteria is likely needed **

- [ ] For CA ORUs:
  - [ ] If OBX-3.3 is null and OBX-3.6 has the value "L", then the OBX segment from the incoming message is converted to an NTE segment.
  - [ ] For the outgoing NTE segment:
    - [ ] OBX-3.5 is moved to NTE-3
    - [ ] OBX-5 is also moved to NTE-3 (depending on OBX-2 value)
    - [ ] OBX-11 is also moved to NTE-3
    - [ ] Possibly other OBX values?

Example: 
Adrenoleukodystrophy deficiency newborn screening comments-discussion|Negative|

## Tasks

- [ ] What tasks need to be done to fulfill the completion criteria?

## Other Notes

- Any other notes to help clarify this task for the team

**Comment by JohnNKing** [comment]
On hold pending communication with CA

**Comment by JohnNKing** [comment]
Still on hold; it's not clear if this approach will be able to be leveraged. Pushing back further into the backlog...

**Comment by JohnNKing** [comment]
Likely to be canceled

**Comment by JohnNKing** [comment]
Canceled in favor of https://github.com/CDCgov/trusted-intermediary/issues/1277
